.buildkite/pipeline.yaml

```diff
@@ -55,7 +55,7 @@ steps:
 
   - label: ":mac: build-static-release.sh (master only)"
     command: .buildkite/build-static-release.sh
-    branches: master
+    #branches: master
```

Comment:
@ptarjan I wil have to edit out the commented branches before merging.

Reply:
👍 feel free to put another commit on this once you see the test run work

Reply:
Done!